tests/run-make/amdgpu-kd/rmake.rs

```diff
@@ -6,13 +6,19 @@
 //@ needs-llvm-components: amdgpu
 //@ needs-rust-lld
 
+use run_make_support::targets::is_windows_gnu;
 use run_make_support::{llvm_readobj, rustc};
 
 fn main() {
+    // rust-lld on gnu targets may spuriously fail with STATUS_HEAP_CORRUPTION (0xc0000374)
+    // To try to mitigate this we pass --threads=1 to the linker.
+    // See #115985.
```

Comment:
Nit (style): can you write this as

```suggestion
    // FIXME(#115985): rust-lld on gnu targets may spuriously fail with STATUS_HEAP_CORRUPTION (0xc0000374)
    // To try to mitigate this we pass --threads=1 to the linker.
```